app/services/course_creation_step_service.rb

```diff
@@ -77,4 +97,52 @@ def further_education_workflow_steps
       confirmation
     ]
   end
+
+  def new_school_direct_workflow_steps
+    %i[
+      courses_list
+      level
+      subjects
+      modern_languages
+      age_range
+      outcome
+      fee_or_salary
+      full_or_part_time
+      location
+      accredited_body
+      applications_open
+      start_date
+      confirmation
+    ]
+  end
+
+  def new_uni_or_scitt_workflow_steps
+    %i[
+      courses_list
+      level
+      subjects
+      modern_languages
+      age_range
+      outcome
+      apprenticeship
+      full_or_part_time
+      location
+      applications_open
```

Comment:
`[*(:entry_requirement if course.provider.recruitment_cycle_year.to_i >= Provider::CHANGES_INTRODUCED_IN_2022_CYCLE)]`

Not fussed